itests-runner/README.md

```diff
@@ -252,7 +252,24 @@ For example:
 }
 ```
 
+### Saving the Cloudify Manager's logs
+Run 
 
+`export CFY_LOGS_PATH_LOCAL=YOUR_LOCAL_PATH && export CFY_LOGS_PATH=YOUR_REMOTE_PATH` 
+
+to save the logs
+to the local path you set.
```

Comment:
What about the remote path? is it necessary ?

Reply:
Yes, otherwise I don't know in advance which path to `scp` from.